.github/workflows/test-application.yaml

```diff
@@ -28,3 +33,130 @@ jobs:
               run: curl "https://cs.symfony.com/download/php-cs-fixer-v2.phar" --output ${{ steps.php-cs-fixer-path.outputs.path }} && chmod +x ${{ steps.php-cs-fixer-path.outputs.path }}
             - name: Run php-cs-fixer
               run: ./${{ steps.php-cs-fixer-path.outputs.path }} fix --dry-run --diff
+
+    test:
+        name: "Run tests with php ${{ matrix.php-version }} (${{ matrix.phpcr-transport }})"
+        runs-on: ubuntu-latest
+
+        env:
+            JACKRABBIT_VERSION: 2.20.2
+            SYMFONY_DEPRECATIONS_HELPER: weak
+            SYMFONY__DATABASE__HOST: 127.0.0.1
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '5.5'
+                      database: mysql
+                      phpcr-transport: doctrinedbal
+                      composer-flags: ''
+                      test-flags: ''
+                      php-extensions: 'ctype, iconv, mysql, gd'
+                      tools: 'composer:v1'
+                      env:
+                          SYMFONY__PHPCR__TRANSPORT: doctrinedbal
+                          SYMFONY__DATABASE__DRIVER: pdo_mysql
+                          SYMFONY__DATABASE__PORT: 3306
+
+                    - php-version: '7.0'
+                      database: mysql
+                      phpcr-transport: doctrinedbal
+                      composer-flags: ''
+                      test-flags: ''
+                      php-extensions: 'ctype, iconv, mysql, imagick'
```

Comment:
think there was not `imagick` in the travis build but not sure if that was intended 😅

Reply:
It might be that it was preinstalled on travis... But the above build is running with gd, might make sense to test both cases anyway :slightly_smiling_face:

Reply:
The `7.4` case also uses `imagick`, so both cases are tested anyway. But i changed `7.0` to `gd` to be consistent with the travis config